src/OpenSage.Game/Logic/Object/Draw/DrawModule.cs

```diff
@@ -30,6 +30,11 @@ public virtual void SetAnimationDuration(LogicFrameSpan frames) { }
 
         public virtual void SetSupplyBoxesRemaining(float boxPercentage) { }
 
+        public virtual void SetTerrainDecal(ObjectDecalType decalType)
+        {
+            // TODO(Port): Implement this for subclasses.
```

Comment:
Up to you, but personally I'd recommend creating a github issue for todos like this so they don't get lost.

Reply:
I actually prefer a TODO in the code (but that's just my personal preference 😄). I don't think there's a danger of it getting lost, it's easily searchable, and it's less overhead than creating an issue.